modules/products/tomcat/src/main/kotlin/com/metalbear/mirrord/products/tomcat/TomcatExecutionListener.kt

```diff
@@ -135,6 +135,11 @@ class TomcatExecutionListener : ExecutionListener {
     }
 
     override fun processStartScheduled(executorId: String, env: ExecutionEnvironment) {
+        val service = env.project.service<MirrordProjectService>()
+        if (!service.enabled) {
+            return
+        }
```

Comment:
Also here - this ignores the `MIRRORD_ACTIVE` environment variable, doesn't it?

Reply:
Same as [here](https://github.com/metalbear-co/mirrord-intellij/pull/292#discussion_r1806691017). I'll remove it